TurkishFilter/sections/specific.txt

```diff
@@ -50,6 +50,7 @@ kocaelidebugun.com###sagAds
 ||turkiyegazetesi.com.tr/images/pageskin/interstitial/libs/ads-mg*.js
 almanilan.com##div.ai-viewports[data-insertion]
 marketingturkiye.com.tr##.ci-ad
+||gundembursa.com/ad-zones.js
```

Comment:
Let's make a rule `/ad-zones.js$domain=` and please add domains from the existing rules

Reply:
`/ad-zones.js` there are three more domains for this rule, should I add them as well?

Reply:
@gork7777 yes